akka-http-core/src/main/mima-filters/10.1.7.backwards.excludes

```diff
@@ -0,0 +1,11 @@
+# HttpMessage new declares methods to represent and set its string representation
```

Comment:
Thanks for being careful about this. I think the change to the 'copy' method is not safe, though: even though there is a default value, so it will be source-compatible, the signature of the public 'copy' method does change, and this would introduce problems with code compiled against the other version of the 'copy' method. I think a way to solve this might be to keep the old signature (but remove the default values), mark it 'deprecated', and add a new one with the additional field. The constructor has the same issue.

Reply:
@raboof I've just applied your suggestion and provided legacy versions of `copy`. Also replaced the default parameter for the constructors by two versions of the constructor, one of them binary  backwards compatible.

The only exclusions are now those for the new methods.